test/unit/core/type_traits/range_iterator_test.cpp

```diff
@@ -71,107 +72,107 @@ TEST(range_and_iterator, value_type_)
 {
     using foreign_iterator = detail::random_access_iterator<std::vector<int>>;
     auto v = std::views::iota(1);
-    using type_list = meta::list<value_type_t<std::vector<int>>,                    // short
-                                 typename value_type<std::vector<int>>::type,       // long
-                                 typename std::vector<int>::value_type,             // member type
-                                 value_type_t<std::vector<int> const>,              // const container
-                                 value_type_t<std::ranges::iterator_t<std::vector<int>>>,        // iterator
-                                 value_type_t<foreign_iterator>,                    // iterator2
-                                 value_type_t<decltype(v)>>;                        // range, no member
-    using comp_list = meta::list<int,
-                                 int,
-                                 int,
-                                 int,
-                                 int,
-                                 int,
-                                 int>;
-
-    expect_same_types<type_list, comp_list>();
+    using type_list_example = type_list<value_type_t<std::vector<int>>,                    // short
+                                        typename value_type<std::vector<int>>::type,       // long
+                                        typename std::vector<int>::value_type,             // member type
+                                        value_type_t<std::vector<int> const>,              // const container
+                                        value_type_t<std::ranges::iterator_t<std::vector<int>>>,        // iterator
+                                        value_type_t<foreign_iterator>,                    // iterator2
+                                        value_type_t<decltype(v)>>;                        // range, no member
+    using comp_list = type_list<int,
```

Comment:
Can you fix the indentation of the comments?
Applies to the whole file

Reply:
You can move the `// iterator` comment to the left, then it should fit, no?

Reply:
Done! I think for one of the comments it exceeds the limit but most of them are fine.